omnigibson/objects/usd_object.py

```diff
@@ -94,22 +121,23 @@ def _load(self):
         if self._encrypted:
             # Create a temporary file to store the decrytped asset, load it, and then delete it
             encrypted_filename = self._usd_path.replace(".usd", ".encrypted.usd")
-            usd_path = self._usd_path.replace(".usd", f".{self.uuid}.usd")
+            decrypted_fd, usd_path = tempfile.mkstemp(os.path.basename(self._usd_path), dir=og.tempdir)
```

Comment:
was this change made because with parallelization there could be a race condition across threads with a non-unique name? (Doesn't this still cause issues on our cluster? IIRC there was some breaking issue with trying to always write to a temp dir, and I think it was because we got OOM problems with our cluster - @cgokmen thoughts?)

Reply:
Writing to the dataset dir is even worse, it is read-only for everyone except me. And yes there is a race condition here. Tempdir should never be running out of space, OS'es job to clean non-open files no?